### ISSUE TITLE
Remove from... Only works by words

### ISSUE THREAD

**Issue by ZZJHONS** [rgriebl/brickstore#553]
The remove from function is good, but it would be better if you could use it to to remove any character not only full words.

PD: A replace function would also be great, I had to use a text editor for mass change some things in Remarks.

Replace could substitute remove as it could be Replace "XXX" with "".

**Comment by paramecie** [comment]
Yes, if remarks are "1234, aaaa", we should be able to remove ", " for instance, or anything.

And then replace would do the trick for everything, agreed.
